addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_channel_target.py

```diff
@@ -24,37 +24,47 @@
 @cached
 def gather_animation_channel_target(channels: typing.Tuple[bpy.types.FCurve],
                                     blender_object: bpy.types.Object,
+                                    bake_bone: typing.Union[str, None],
+                                    bake_channel: typing.Union[str, None],
                                     export_settings
                                     ) -> gltf2_io.AnimationChannelTarget:
```

Comment:
Hm, it seems worrying to see a `@cached` function that isn't taking any parameters related to the current NLA track or Action name. This seems like a probable cause of https://github.com/KhronosGroup/glTF-Blender-IO/issues/451 and https://github.com/KhronosGroup/glTF-Blender-IO/issues/461?

Reply:
Yes, you're probably right.
#461 will be my next work, after #426 and #460 will be merged